.eslintrc.cjs

```diff
@@ -1,6 +1,10 @@
 module.exports = {
   parser: "@typescript-eslint/parser", // Specifies the ESLint parser
-  plugins: ["turbo"],
```

Comment:
The fact that this was here was a bug, as the turbo rule wasn't even being used by the codebase.